docs/Orchestration/alternative-schedulers.md

```diff
@@ -1,10 +1,157 @@
 ---
 title: Alternative Schedulers
-draft: true
+id: alternative-schedulers
+description: Support for Alternative Orchestration Solutions
+tags:
+  - jobs
+  - deployment
+  - orchestration
+  - spark-submit
+  - runtime config
 ---
 
-:::caution 🚧 Work in Progress 🚧
+## Basic Spark Submit
 
```

Comment:
Consider adding a short intro sentence under ## Basic Spark Submit. You can simply state that the following sections contain Scala Spark, PySpark, and runtime configuration variables to use with custom orchestration solutions.